src/services/projectService.ts

```diff
@@ -53,6 +53,10 @@ export default class ProjectService implements IProjectService {
                     project.id = shortid.generate();
                 }
 
+                if (project.version !== "v2") {
```

Comment:
We should pull the exact version info from the executing app.

Reply:
We should also add version to the `IProject` interface as required.

Reply:
What do you mean by your first comment? And if it is required, where would we set the version when the project is created?

Reply:
For example, if we release a new version `2.1.5` as defined withing the `package.json` then our app should use this version to auto-version the project file anytime it is saved.